tripal_chado/src/Plugin/TripalStorage/ChadoStorage.php

```diff
@@ -1093,35 +1122,43 @@ protected function handleReadValue(array $context, StoragePropertyValue $prop_va
   }
 
   /**
-   * Takes a path string for a field property and converts it to an array structure.
+   * Parses a field property path string into an array structure.
    *
    * @param string $field_name
    *   The name of the field.
    * @param string $base_table
    *   The name of the base table for thie field.
    * @param mixed $path
-   *   A string continaining the path.  Note: this is a recursive function and on
-   *   recursive calls this variable will be n array. Hence, the type is "mixed".*
+   *   A string continaining the path. Note: this is a recursive function and on
+   *   recursive calls this variable will be an array.
    * @param array $aliases
    *   Optional. The list of table aliases provdied by the `table_alias_mapping`
-   *   argument of a field.  If this variable is an empty array then the function
-   *   will use the table name provided in the path.
+   *   argument of a field.  If this variable is an empty array then the
+   *   function will use the table name provided in the path.
    * @param string $as
-   *   An alias to be used for the Chado table column that contains the value. This
-   *   argument will rename the column.
+   *   An alias to be used for the Chado table column that contains the value.
+   *   This argument will rename the column.
    * @param string $full_path
-   *   This argument is used by recursion to build the string path for each level.
-   *   It should not be set by the callee.
-   * @return array
+   *   This argument is used by recursion to build the string path for each
+   *   level. It should not be set by the callee.
    *
+   * @return array
+   *   An array describing the path.
    */
-  protected function parsePath(string $field_name, string $base_table, mixed $path,
-      array $aliases = [], string $as = '', string $full_path = '') {
+  protected function parsePath(
+    string $field_name,
+    string $base_table,
+    mixed $path,
+    array $aliases = [],
+    string $as = '',
+    string $full_path = '',
+  ) {
 
     // If the path is a string then split it.
     $path_arr = [];
     if (is_string($path)) {
-      // For sanity sake, remove any trailing semicolons that might be there by accident.
+      // For sanity sake, remove any trailing semicolons that might be there
```

Comment:
```suggestion
      // For sanity's sake, remove any trailing semicolons that might be there
```
can't help fixing 🙃...